Source/Filter Effects/feGaussianBlur/SvgGaussianBlur.cs

```diff
@@ -13,55 +13,64 @@ public enum BlurType
     [SvgElement("feGaussianBlur")]
     public class SvgGaussianBlur : SvgFilterPrimitive
     {
-        private float _stdDeviation;
+        private float _stdDeviationX = float.NaN;
+        private float _stdDeviationY = float.NaN;
+        private bool _isPrecalculated = false;
         private int[] _kernel;
         private int _kernelSum;
         private int[,] _multable;
 
-        public SvgGaussianBlur()
-            : this(1f, BlurType.Both)
+        private void PreCalculate()
         {
-        }
+            float stdDeviationX = 0f;
+            float stdDeviationY = 0f;
 
-        public SvgGaussianBlur(float stdDeviation)
-            : this(stdDeviation, BlurType.Both)
-        {
-        }
+            if (StdDeviation.Count == 1)
+            {
+                stdDeviationX = StdDeviation[0];
+                stdDeviationY = stdDeviationX;
+            }
+            else if (StdDeviation.Count == 2)
+            {
+                stdDeviationX = StdDeviation[0];
+                stdDeviationY = StdDeviation[1];
+            }
 
-        public SvgGaussianBlur(float stdDeviation, BlurType blurType)
-            : base()
-        {
-            _stdDeviation = stdDeviation;
-            BlurType = blurType;
-            PreCalculate();
-        }
+            if (_isPrecalculated && _stdDeviationX == stdDeviationX && _stdDeviationY == stdDeviationY)

```

Comment:
Is it unnecessary to check _isPrecalculated ?
This method executes precalculate when XY is changed, regardless of _isPrecalculated, right ?

Reply:
This method only needs to be executed only when XY changes.

Reply:
So, change.

```cs
if (_stdDeviationX == stdDeviationX && _stdDeviationY == stdDeviationY)
```

Reply:
Pushed the change.